### DISCUSSION TITLE
Github Graduation PR check

### DISCUSSION THREAD

**Discussion by j-alicia-long** [community/community#17774] [GitHub Education]
My PR keeps failing the automated linter, not sure what I'm missing https://github.com/education/GitHubGraduation-2022/pull/2107

**Comment by sredas** [comment]
I have the same question! https://github.com/education/GitHubGraduation-2022/pull/2385

Unfortunately haven't found an answer yet

**Reply by sredas** [reply]
I think the issue was that we both included colons in our quotes. I just resubmitted mine a couple minutes ago, hopefully it goes through!

**Reply by j-alicia-long** [reply]
Ah that was it! thanks :)

**Comment by mlarasusan** [comment]
Hey @sredas & @j-alicia-long, 

You can create an [Issue](https://github.com/education/GitHubGraduation-2022/issues) in the graduation repository and the team will be able to help you with your pull request [✨](https://emojipedia.org/sparkles/) share the solution with us later [🙌](https://emojipedia.org/raising-hands/)
